static/website/templates/get-question.html

```diff
@@ -26,7 +26,6 @@ <h5>Title</h5>
             <input class="form-control" type="text">
             <h5>Question</h5>
         </div>
-        <a class="delete-question btn btn-xs btn-info" data-qid="{{ question.id }}" href="#">Delete</a>
```

Comment:
does this need to be removed, doesn't look like part of this pull-request. @kirti3192

Reply:
This button was made there to check whether we can delete the cross user questions or not. 
That was just for testing, so have to remove it.

Reply:
@theskumar : can we merge this?